### ISSUE TITLE
v0.7.1 on GitHub repo, v0.8.0 on PyPI

### ISSUE THREAD

**Issue by vst** [sbdchd/djangorestframework-types#31]
Latest release on the GitHub repository seems to be v0.7.1. On PyPI, we have v0.8.0.

Does that matter?

**Comment by sbdchd** [comment]
oh hmm, maybe I pushed something to pypi and didn't merge it

**Comment by vst** [comment]
Probably. There is only https://github.com/sbdchd/djangorestframework-types/commit/dc69c7d5fafaac1d5f2c2acc95fa569e6ba6f9e7. You may wish to rebase master if you think that it is not a bad idea (and you can find the commits).

**Comment by dolfandringa** [comment]
Could you please merge your code back in here? There is still a version discrepancy between this repo and pypi.

**Comment by dolfandringa** [comment]
I don't know what went on with that version, but it also reverts fixes from 0.7.1, so I would suggest pinning your versions to 0.7.1

**Comment by simensol** [comment]
I have `v0.8.0` installed through PyPi.

The following code:

```py
from rest_framework.permissions import BasePermission

class CanChangeChild(BasePermission):
    ...

@action(
    detail=True,
    methods=["patch"],
    serializer_class=UpdateProfilePictureSerializer,
    permission_classes=[CanChangeChild],
)
def update_profile_picture(self, request, pk=None):
    ...
```

raised the following error from Pylance:

> Argument of type "list[type[CanChangeChild]]" cannot be assigned to parameter "permission_classes" of type "_PermClassesParam" in function "action" Type "CanChangeChild" is not assignable to type "type[BasePermission]"

After checking the types, I found this in `rest_framework-stubs/decorators.pyi`

```py
_PermClassesParam = Sequence[Type[BasePermission]]
```

However, the issue seems corrected in the master branch:

https://github.com/sbdchd/djangorestframework-types/blob/bd865b5301b8b3df71aefe14401704746770eb20/rest_framework-stubs/decorators.pyi#L93

The fix was made in #40 on Jul 25, 2023:

https://github.com/sbdchd/djangorestframework-types/commit/2a445463313bced492b7e4401d964f7855fac33c#diff-311444d4598e5dbe12bf1c0dd49f2afbbb13dbdff781bedabeb638d5f99e4f94L93-L94

I tried pinning to `v0.7.1` as suggested by @dolfandringa, but the fix doesn’t seem included in that version either. The only solution was installing from the `master` branch, which is not ideal.

Would it be possible, @sbdchd, to publish the latest `master` as a new version to PyPi?

**Comment by sbdchd** [comment]
should be fixed!
